osc.c

```diff
@@ -1230,7 +1230,7 @@ static gboolean capture_process(void *data)
 			if (offset / (off_t)sizeof(gfloat) < info->offset / 4) {
 				offset = 0;
 			} else if (offset) {
-				offset -= info->offset * sizeof(gfloat) / 4;
+				offset -= (info->offset / 4) * sizeof(gfloat);
 				for (i = 0; i < nb_channels; i++) {
```

Comment:
the commit could use a bit more description about some of the intricate details of this fix

Reply:
i'm referring to commit `osc: Fix trigger corrupting all data when capturing an odd nb of samples`
Github reports them in a weird way

Reply:
Done